src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
+@import "./utils/variables";
+@import "./blocks/container.scss";
+
 body {
   margin: 0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  color: #eee;
```

Comment:
It should be white. Probably that's the reason why tests fail

Reply:
I have fixed it, now all my tests are passed. thank you